lib/repl.js

```diff
@@ -409,7 +408,7 @@ function REPLServer(prompt,
 
     // Check to see if a REPL keyword was used. If it returns true,
     // display next prompt and return.
-    if (cmd && cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
+    if (cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
```

Comment:
The existing code does not protect against non-strings. It only protects against falsy values. 

You can't send a non-string without messing with `REPLServer`. I'll see about writing a test to make sure we are failing the same way when someone does that.